plugin/src/functionalTest/kotlin/io/qalipsis/gradle/cloud/CloudRunQalipsisTaskFunctionalTest.kt

```diff
@@ -0,0 +1,655 @@
+                            "timeout": "PT1H30M",
+                            "hardTimeout": false,
+                            "scenarios": {
+                                "My scenario 1": {
+                                    "minionsCount": 7,
+                                    "zones": {
+                                        "CH": 45,
+                                        "AS": 55
+                                    },
+                                    "executionProfile": {
+                                        "profile": "PROGRESSING_VOLUME",
+                                        "periodMs": 11,
+                                        "minionsCountProLaunchAtStart": 23,
+                                        "multiplier": 2.0,
+                                        "maxMinionsCountProLaunch": 7
+                                    }
+                                }
+                            }
+                        }
+                        """.trimIndent()
+                    )
+                )
+        ).respond(
+            response()
+                .withStatusCode(200)
+                .withHeader(Header("Content-Type", "application/json; charset=utf-8"))
+                .withBody(
+                    """
+                        {
+                            "version": "2025-04-28T13:09:43.533Z",
+                            "key": "Dummy campaign configuration",
+                            "creation": "2025-04-28T13:09:43.533Z",
+                            "name": "Dummy campaign configuration",
+                            "speedFactor": 2.0,
+                            "scheduledMinions": null,
+                            "hardTimeout": "2025-04-28T13:09:44.533Z",
+                            "start": null,
+                            "end": null,
+                            "status": "IN_PROGRESS",
+                            "configurerName": null,
+                            "aborterName": "qalipsis-ten",
+                            "scenarios": [
+                               {
+                                  "name": "My scenario 1",                        
+                                  "minionsCount": 7,
+                                  "version": "2025-04-28T13:09:43.533Z"
+                               }
+                            ],
+                            "zones": ["CH", "AS"]
+                        }
+                    """.trimIndent()
+                )
+        )
+
+        // When
+        val runner = GradleRunner.create()
+            .forwardOutput()
+            .withPluginClasspath()
+            .withProjectDir(tempFolder.toFile())
+            .withDebug(true)
+            .withArguments("qalipsisRunCampaign")
+        val result = runner.build()
+
+        // Then
+        result.output shouldContain """
+            Task :qalipsisRunCampaign
+            Starting a new campaign with name Dummy campaign configuration
```

Comment:
It would be great if you could also display something like this in the Gradle console:
```
You can retrieve the details of the campaign from the link <http-root>/campaigns/<campaignKey>
```

Where you replace the placeholders by the real values. In the example of the test, the complete URL will be something like:
`http://localhost:36516/api/campaigns/r0g0u1piyI` if 36516 is the port of the mock server.